### ISSUE TITLE
Device not implemented: HmIP-HDM1

### ISSUE THREAD

**Issue by ranjahz** [marcsowen/homebridge-homematicip#185]
Seems like HmIP-HDM1 [Hunter Douglas Blinds] is currently not supported for Homekit/Homemtic IP combination.
Is there a plan to implement this type of device?

THX

**Comment by marcsowen** [comment]
Yes, I looked at it, but those shades seem to work a bit differently. It needs a new implementation. I will try...

**Comment by ranjahz** [comment]
THX mate, really looking forward for this one

**Comment by vojj** [comment]
Hi marc, i bought a rollo as device type hdm1. Did you have any progess to support this device type. Maybe i can help you to implement this?
best regards

**Comment by st32a1y** [comment]
Hello Marc,
Thank you for doing this. As the others I'd be really grateful for being able to implement those shades into HomeKit.
Happy New Year btw
Best regards

**Comment by Scotti010** [comment]
Moin Marc, 
ich probier's mal auf deutsch ;-) Die Anbindung des Lichtsensors (SLO) funktioniert super. Wenn jetzt noch HDM1 dazu käme, wäre es die perfekte Lösung! Wie können wir Dich unterstützen? LG aus BS! Dagmar & Frank

**Comment by decodrr** [comment]
Hi @marcsowen et al.
I joined the HmIP-HDM1(Rollo) club today and found out that the device was not yet supported. 
Any chance we can get this working this summer? :-) 
Thanks a lot!

**Comment by marcsowen** [comment]
HmIP-HDM1 support was added in version 2.2.0.
